src/SFA.DAS.ApprenticeCommitments.Api/Controllers/QueryForwardingController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ApprenticeCommitments.Application.Services;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Api.Controllers
+{
+    [Controller]
+    public class QueryForwardingController : ControllerBase
+    {
+        private readonly GenericInnerApiQueryClient client;
+
+        public QueryForwardingController(GenericInnerApiQueryClient client)
+        {
+            this.client = client;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> ForwardToInnerApi()
+        {
+            var innerResponse = await client.GetAsync(HttpContext.Request.Path);
+
```

Comment:
My first concern with this pattern is all the GETS to outer API will be passed to the inner API (this might be limiting).